### DISCUSSION TITLE
Option to make BTTV badge(s) invisible

### DISCUSSION THREAD

**Discussion by thekillernacho** [night/betterttv#6267] [Ideas]
I use BTTV with an onstream chat window that's fairly narrow and have most of twitch's badges disabled on it via another extension. I noticed BTTV's recent addition of the BTTV Pro Subscriber badge and wanted to disable it from appearing on my local chat but I do not see an option in BTTV to disable this. (I know I can disable _mine_ on the BTTV site, but Id like to disable other people's on my own chat window, too)

Please add functionality to disable seeing this BTTV badge (and any future badges added and rendered by BTTV in the future). Thank you.

**Comment by kozaka-tv** [comment]
I think this is done already. You can turn this off under the BetterTTV Pro page.
